### PR TITLE
modify some text

### PR DESCRIPTION
**Pull request by FranktheTank123**
this is a pull requires someone to review

### CODE REVIEW

**Review by TarekMohammad1 — APPROVED** [review]

**Review by LynnHunt123 — APPROVED** [review]

**Review by tnjiit — CHANGES_REQUESTED** [review]
Decline.

File: Lectures/Lecture 1/readme.md [hunk 1]
```diff
@@ -55,13 +55,14 @@
...
   - Pull request 
     - [Why do we need a pull request?](https://co-learning.eu/2017/10/04/why-and-how-do-we-use-pull-request/)
-    - create a branch and make some changes
-    - branch naming practice
-        - `hw_<hw_number>_<firstname>_<lastname>`, e.g.: `hw_1_frank_xia`
-    - push that branch to remote: `git push --set-upstream origin <branch_name>`
-    - create a pull request
-    - assign reviewers
-    - merge pull request
+    - things you need to do for pull request
```
Comment (FranktheTank123) [inline Lectures/Lecture 1/readme.md:58]: there is a typo

**Review by krika1611 — APPROVED** [review]

File: Lectures/Lecture 1/readme.md [hunk 2]
```diff
@@ -55,13 +55,14 @@
...
+      - create a branch and make some changes 
+      - push that branch to remote: `git push --set-upstream origin <branch_name>`
+      - branch naming practice
+          - `hw_<hw_number>_<firstname>_<lastname>`, e.g.: `hw_1_frank_xia`
+      - create a pull request
+      - assign reviewers
+      - merge pull request
```
Comment (FranktheTank123) [inline Lectures/Lecture 1/readme.md:65]: please use a different wording

**Review by himabindualwal — COMMENTED** [review]
Looks good

**Pull request merged**